### PR TITLE
Add reference to a TypeScript typing definition

### PR DESCRIPTION
**Pull request by hydroper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,6 +266,10 @@ will be generated.
 * Initial tests are scaffolded from [yueliang][yueliang] and then manually checked for error.
 * Much of the code is based on [LuaMinify][luaminify], the [Lua][lua] source and [Esprima][esprima]. All awesome projects.
 
+## TypeScript typing definitions
+
+* [luaparse-std](https://github.com/hydroper/luaparse-tsd)
```
Comment (oxyc) [inline README.md]: `luaparse-tsd` no?

Comment (hydroper) [inline README.md]: I think tsd means typescript definition...

Comment (oxyc) [inline README.md]: I believe you typod the name as `luaparse-std` rather than `luaparse-tsd` which is the name of the repo.

**Comment by oxyc** [comment]
Also would you mind moving it under _Projects using luaparse_, maybe renaming it _Projects using/extending luaparse_, and I'll merge it.

File: README.md (hunk 1 above)
Comment (hydroper) [inline README.md]: It's really (tsd)[https://github.com/DefinitelyTyped/tsd], I think you're confusing it with C++?

**Comment by hydroper** [comment]
0f course!

File: README.md (hunk 1 above)
Comment (oxyc) [inline README.md]: I understand, but you commit says:

```
[luaparse-std](https://github.com/hydroper/luaparse-tsd)
          ~~~
```

:)

Comment (hydroper) [inline README.md]: Done, I modified it

**Comment by oxyc** [comment]
Perfect, thanks!

**Comment by hydroper** [comment]
Thank you for the great parser :D

**Pull request merged**